.github/workflows/test.yml

```diff
@@ -32,11 +32,7 @@ jobs:
       matrix:
         script:
           - scripts/metrics/discovery_tables.py
-          - scripts/model_figs/rolling_mae_vs_hull_dist_models.py
-          - scripts/model_figs/cumulative_metrics.py
-          - scripts/rolling_mae_vs_hull_dist.py
-          - scripts/hist_classified_stable_vs_hull_dist.py
-          - scripts/model_figs/hist_classified_stable_vs_hull_dist_models.py
+          - scripts/model_figs/run_all.py
```

Comment:
the thing with `run_all.py` is it runs all scripts in sequence whereas the GHA matrix was running them in parallel. since each script runs as its own process, it needs to reload the model preds in each script which makes it quite slow

Reply:
it took 3m30s so it only slows down the overall CI by like 90s. I would say this is better as it tests more, tests all the things we care about and less maintenance but can revert

Reply:
sounds fine, just wanted to point out